app/src/main/java/com/tvtracker/dto/MediaItem.kt

```diff
@@ -2,11 +2,13 @@ package com.tvtracker.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class MediaItem(@SerializedName("imdbID")      var id: String = "",
-                     @SerializedName("Title")       var title: String = "",
-                     @SerializedName("Year")        var year: String = "",
-                     @SerializedName("Type")        var type: String = "",
-                     @SerializedName("Poster")      var imageUrl: String = "",
-                     @SerializedName("platform")    var platform: String = "",
-                     @SerializedName("description") var description: String = "",
-                     @SerializedName("watched")     var watched: Boolean = false)
+data class MediaItem(
+    @SerializedName("imdbID")      var id: String = "",
+    @SerializedName("Title")       var title: String = "",
```

Comment:
Restructured dto signatures to proper coding conventions for readability.
https://developer.android.com/kotlin/style-guide#formatting